src/components/price/price.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from './price.module.scss';
+
+export interface PriceProps {
+    fullPrice: string | undefined;
+    price: string | undefined;
```

Comment:
call it `discoutedPrice` or `currentPrice`

Reply:
or actualPrice